client/my-sites/plugins/main.jsx

```diff
@@ -112,6 +129,32 @@ export class PluginsMain extends Component {
 		this.unsubscribe();
 	}
 
+	resetBreadcrumbs() {
+		const { selectedSiteSlug, search } = this.props;
+
+		this.props.updateBreadcrumbs( [
+			{
+				label: this.props.translate( 'Plugins' ),
+				href: `/plugins/${ selectedSiteSlug || '' }`,
+				helpBubble: this.props.translate(
+					'Add new functionality and integrations to your site with plugins.'
+				),
+			},
+			{
+				label: this.props.translate( 'Installed Plugins' ),
+				href: `/plugins/manage/${ selectedSiteSlug || '' }`,
+			},
+		] );
+
+		if ( search ) {
+			this.props.appendBreadcrumb( {
+				label: this.props.translate( 'Search Results' ),
+				href: `/plugins/${ selectedSiteSlug || '' }?s=${ search }`,
```

Comment:
Ack, sorry. I just noticed an issue. If you search on `/plugins/manage`, click on a plugin from the search results, and then click "Search Results" in the breadcrumbs, it takes you to the wrong search results. I think we need to add `/manage/` to the `href` here, and on line 110, to fix that.